### DISCUSSION TITLE
Score not appearing in lazer

### DISCUSSION THREAD

**Discussion by PPenn4** [ppy/osu#29943] [Q&A] [answered]
I have a score on https://osu.ppy.sh/beatmapsets/1412517#osu/2911715, however, it does not appear on the lazer version of the website or within the lazer client. 

Edit: Score was set in standard

First image shows the score on the standard version of the website.
![image](https://github.com/user-attachments/assets/0ea5fa4f-27ed-4c03-897a-38a144e01d20)

Second image shows the the map on the lazer version of the website.
![image](https://github.com/user-attachments/assets/2a09eb6d-2136-42cb-bb94-aec1c98702e4)

Third image to show score is not present in osu lazer.
![osu_2024-09-20_15-18-00](https://github.com/user-attachments/assets/5cd53588-4c82-41bd-857d-e98b38622dd8)

The score can be found on the lazer website through extensions (osuplus)
![image](https://github.com/user-attachments/assets/088f94f9-265e-458c-a4d3-5c84366e9fbc)

This is what searching for the score with osuplus looks like on the standard website
![image](https://github.com/user-attachments/assets/8b83c923-65b9-4240-8dc1-acb09fb2c872)

**Comment by cihe13375** [comment]
The 2nd image is weird because the leaderboard below is not really in lazer mode (it should display the classic mode icon, also the max score should be around 1000k)

I suggest try again with osuplus disabled.

**Reply by PPenn4** [reply]
Leaderboard with osuplus disabled
![image](https://github.com/user-attachments/assets/cffb3c7d-52f2-4371-b2df-99b8726691d3)


The issue is also present within the beatmap listing within osu lazer itself: 

This beatmap (https://osu.ppy.sh/beatmapsets/384772#osu/930249) displays my standard score as intended
![osu_2024-09-23_13-25-34](https://github.com/user-attachments/assets/38a4d616-2e22-4d76-80bf-414f68d7340c)

But Toyota Corolla still fails
![image](https://github.com/user-attachments/assets/f5e4c3ad-c732-4568-ac9a-76562ac5160f)


The score does in fact exist in osu lazer, but my only ways of accessing it are through pinning it or opening the score details while in the standard website.
![image](https://github.com/user-attachments/assets/ffb0529f-e425-4f04-9cf2-22c8e3e22905)


I have already tried deleting and redownloading Toyota Corolla and it did not fix the issue. Importing the replay also did not do anything other than add the letter grade I got to the difficulty selection.
![osu_2024-09-23_13-41-28](https://github.com/user-attachments/assets/d1fe9015-84a6-4701-94ba-564acdb11c22)

**Reply by peppy** [reply]
@PPenn4 can you check if it's any better now?

**Reply by PPenn4** [reply]
I currently still cannot see the score within the game and the lazer webpage remains the same.

Lazer webpage within client:
![osu_2024-10-05_14-17-20](https://github.com/user-attachments/assets/085cab08-ad55-47ca-9511-7057b7a3e6fd)

Ingame leaderboard:
![osu_2024-10-05_14-18-02](https://github.com/user-attachments/assets/e191e774-02fa-4330-9297-5aec226de1a8)

Lazer webpage in web browser - For reference, turning on classic scoring does not change anything other than the score numbers
![Screenshot 2024-10-05 141623](https://github.com/user-attachments/assets/548735c9-1859-4ee1-b2bf-37e3817565c8)

**Reply by PPenn4** [reply]
Overriding the score does not appear to have done anything either
Local leaderboard - the classic replay was downloaded from webpage
![osu_2024-10-05_14-30-58](https://github.com/user-attachments/assets/628beedc-7a12-4989-8b8d-ebdca1369332)

Global leaderboard
![osu_2024-10-05_14-31-03](https://github.com/user-attachments/assets/2f25e98d-3e96-4196-8fe1-c8a29e03bc9f)

The score does in fact exist and did get uploaded
![Screenshot 2024-10-05 143300](https://github.com/user-attachments/assets/a5346a9f-bcbc-4787-a4a3-165e2e46e932)

But it still remains unavailable from the beatmap listing
![image](https://github.com/user-attachments/assets/4e6c0b5e-c491-413b-a835-383470038d0b)

And due to the nature of lazer scores, it is not available on the classic website. The only way to access the score from the webpage is from my profile (https://osu.ppy.sh/users/14680807), where it is currently available under Recent Plays (24h), and I have pinned the score for future viewing after 24 hours pass. 

Edit: 
The score on Seasons (https://osu.ppy.sh/beatmapsets/449873#osu/965243), which is also in the recent plays screenshot, can be seen on the lazer webpage, so the issue appears to be limited to Toyota Corolla.

**Reply by nanaya** [reply, marked as answer]
there's a deleted score stuck in the index and broke things. This specific instance should be fixed now.
